### PR TITLE
feat(timeline): remove shim and fallbacks

### PR DESCRIPTION
**Pull request by valentin-mladenov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [x] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: CDE-1223

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by valentin-mladenov** [comment]
Follow of #865

File: projects/angular/src/timeline/_variables.timeline.scss [hunk 1]
```diff
@@ -3,63 +3,23 @@
...
-$clr-timeline-gutter-width: variables.$clr_baselineRem_0_5;
-
-// timeline step (horizontal)
-$clr-timeline-step-min-width: variables.$clr_baselineRem_7_29167;
-
-// header (timestamp)
-$clr-timeline-step-header-font-size: variables.$clr-p2-font-size;
-$clr-timeline-step-header-line-height: variables.$clr_baselineRem_0_75;
-$clr-timeline-step-header-padding-bottom: variables.$clr_baselineRem_8px;
+$clr-timeline-icon-size: #{tokens.$cds-global-space-11};
```
Comment (mivaylo) [inline projects/angular/src/timeline/_variables.timeline.scss:19]: I am not sure why we have these as variables in the variables file - maybe use the cds tokens inside the timeline.clarity file?

**Review by mivaylo — APPROVED** [review]

File: projects/angular/src/timeline/_variables.timeline.scss [hunk 2]
```diff
@@ -3,63 +3,23 @@
...
-$clr-timeline-vertical-step-header-top-padding: variables.$clr_baselineRem_0_25;
-$clr-timeline-vertical-step-header-right-padding: variables.$clr_baselineRem_0_5;
-
-$clr-timeline-vertical-step-body-top-padding: variables.$clr_baselineRem_0_25;
-$clr-timeline-vertical-step-body-left-padding: variables.$clr_baselineRem_0_5;
-
-$clr-timeline-vertical-min-width: variables.$clr_baselineRem_13_33;
-$clr-timeline-vertical-step-min-width: variables.$clr_baselineRem_7_33;
-$clr-timeline-vertical-gutter-width: variables.$clr_baselineRem_1_5;
+$clr-timeline-step-title-font-weight: var(--clr-timeline-step-title-font-weight);
```
[the remark this answers is not in the document]
Comment (valentin-mladenov) [reply projects/angular/src/timeline/_variables.timeline.scss:25]: `$clr-timeline-step-title-font-weight` variable is for internal purpose only. 
`--clr-timeline-step-title-font-weight` property is exposed for override.

File: projects/angular/src/timeline/_variables.timeline.scss (hunk 1 above)
Comment (valentin-mladenov) [reply projects/angular/src/timeline/_variables.timeline.scss:19]: `$clr-timeline-icon-size`, `$clr-timeline-line-thickness` and `$clr-timeline-default-icon-size` are variables for internal use only and to see how some rues are calculated.

**Pull request merged**